content/influxdb/cloud/reference/release-notes/influx-cli.md

```diff
@@ -8,6 +8,20 @@ menu:
     name: influx CLI 
 ---
 
+## v2.1.1 [unreleased]
+
+## Go Version 
+
+This release upgrades the project to `go` version 1.17.
+
+## Bug Fixes 
+
+- Fix shell completion for top-level `influx` commands.
+- Make global `--http-debug` flag visible in help text.
```

Comment:
Do you want/need to add this flag to the detailed reference docs for each CLI command?

Reply:
Yes. we should consider adding it here. that can be done in another PR. https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/#flags